From ea6256a346adbd65ac1d646d716e7b51ea0b9232 Mon Sep 17 00:00:00 2001
From: Ville-Pekka Vainio <vpvainio@iki.fi>
Date: Fri, 9 Jun 2023 12:00:27 +0300
Subject: [PATCH] FlateDecode does not mean password protected

Have the patch apply against file-5.39 in EL9
Upstream bug: https://bugs.astron.com/view.php?id=275
Upstream commit: https://github.com/file/file/commit/239852073ad3a5ba5cf62b21e95565f122869f6a
Original author: pwinckles

Signed-off-by: Ville-Pekka Vainio <vpvainio@iki.fi>
---
 magic/Magdir/pdf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/magic/Magdir/pdf b/magic/Magdir/pdf
index e386f454..7aa7a24d 100644
--- a/magic/Magdir/pdf
+++ b/magic/Magdir/pdf
@@ -5,7 +5,7 @@
 #
 
 0	name	pdf
->8	search/512	/Filter/FlateDecode/	(password protected)
+>8	search/512	/Filter/FlateDecode/	(zip deflate encoded)
 
 0	string		%PDF-		PDF document
 !:mime	application/pdf
-- 
2.40.1

